packages/frontend/src/components/profile_pic.tsx

```diff
@@ -0,0 +1,78 @@
+import type { Sport } from "@esp-group-one/types";
+import { useState } from "react";
```

Comment:
Linter is failing: https://github.com/ESP-Number-One/MatchyoSports/actions/runs/8032227158/job/21941388728?pr=56